pyth_observer/events.py

```diff
@@ -234,13 +242,35 @@ def get_event_details(self) -> Tuple[str, List[str]]:
             f"Aggregate last slot: {self.price.aggregate.slot}"
             f"Published last slot: {self.publisher_latest.slot}"
         )
-
         return title, details
 
-
 # Price Account events
 
 
+class LatePublishingOrUnknown(PriceAccountValidationEvent):
```

Comment:
this could be like "PriceFeedOffline" or something. Same for the error code. 

Also for the comment, it might be good to explain what the alert is supposed to do. I think this alert is supposed to fire when a price feed should be updating, but isn't. You can then explain how that check is operationalized (the slot difference / status) confidiont.